Library/Homebrew/cask/audit.rb

```diff
@@ -487,11 +491,14 @@ def audit_signing
 
           next if result.success?
 
-          add_error <<~EOS, location: cask.url.location, strict_only: true
+          add_error <<~EOS, location: cask.url.location
             Signature verification failed:
             #{result.merged_output}
             macOS on ARM requires software to be signed.
             Please contact the upstream developer to let them know they should sign and notarize their software.
+            Alternatively, under some criteria this can be bypassed by adding caveat "unsigned_artifact".
```

Comment:
```suggestion
            Users can be notified by adding caveat "unsigned_artifact".
```

Reply:
Or we should warn for existing casks and throw the error on new casks?

Reply:
> Or we should warn for existing casks and throw the error on new casks?

We've removed warnings from `brew audit` because it's confusing. I think we should figure out some middle ground, though, so that we do flag these sorts of things as a problem when people are running `brew audit`.

I think perhaps we're at the stage where, yeh, that should mean this is run every time and we start adjusting casks that are not compliant even on version bumps.